src/com/owncloud/android/lib/common/operations/RemoteOperationResult.java

```diff
@@ -119,7 +119,8 @@
         DELAYED_FOR_WIFI,
         LOCAL_FILE_NOT_FOUND,
         SERVICE_UNAVAILABLE,
-        SPECIFIC_SERVICE_UNAVAILABLE
+        SPECIFIC_SERVICE_UNAVAILABLE,
+        SPECIFIC_UNSUPPORTED_MEDIA_TYPE
```

Comment:
There is no need for the ```SPECIFIC_``` prefix. There is no other 'non-specific' ```UNSUPPORTED_MEDIA_TYPE``` that needs to be distinguished. Let's keep it simple as ```UNSUPPORTED_MEDIA_TYPE```